README.md

```diff
@@ -1,135 +1,239 @@
-| GET    | `/users/org/:userId`    | owners, supervisors | Returns all users for an organization.             |
-| GET    | `/users/:userId`        | owners, supervisors | Returns info for a single user.                    |
-| POST   | `/users/register/owner` | none                | Creates a new user as owner of a new organization. |
-| PUT    | `/users/:userId`        | owners, supervisors |                                                    |
-| DELETE | `/users/:userId`        | owners, supervisors |                                                    |
+```
+  type Event {
+    id: ID!
+    Date: String!
+    Start_Time: String!
+    End_Time: String
+    Title: String!
+    Description: String!
+    Photo: String!
+    category_id: Int!
+    user_id: Int!
+    Modifiers: String!
+    Address: String!
+    Latitude: Float!
+    Longitude: Float!
+  }
+```
 
-# Data Model
+```
+  input NewEventInput {
+    id: ID
+    Date: String!
+    Start_Time: String!
+    End_Time: String
+    Title: String!
+    Description: String!
+    user_id: Int!
+    Photo: String
+    category_id: Int!
+    Modifiers: String
+    Address: String!
+    Latitude: Float!
+    Longitude: Float!
+  }
+```
 
-🚫This is just an example. Replace this with your data model
+```
+  input UpdateEventInput {
+    id: ID
+    Date: String
+    Start_Time: String
+    End_Time: String
+    Title: String
+    Description: String
+    Photo: String
+    category_id: Int
+    user_id: Int
+    Modifiers: String
+    Address: String
+    Latitude: Float
+    Longitude: Float
+  }
+```
 
-#### 2️⃣ ORGANIZATIONS
+#### Category Type and Inputs
 
 ---
 
 ```
-{
-  id: UUID
-  name: STRING
-  industry: STRING
-  paid: BOOLEAN
-  customer_id: STRING
-  subscription_id: STRING
-}
+  type Category {
+    id: ID!
+    Category: String!
+  }
 ```
 
-#### USERS
-
----
-
 ```
-{
-  id: UUID
-  organization_id: UUID foreign key in ORGANIZATIONS table
-  first_name: STRING
-  last_name: STRING
-  role: STRING [ 'owner', 'supervisor', 'employee' ]
-  email: STRING
-  phone: STRING
-  cal_visit: BOOLEAN
-  emp_visit: BOOLEAN
-  emailpref: BOOLEAN
-  phonepref: BOOLEAN
-}
+  input NewCategoryInput {
+    id: ID
+    Category: String!
+  }
 ```
 
 ## 2️⃣ Actions
 
-🚫 This is an example, replace this with the actions that pertain to your backend
-
-`getOrgs()` -> Returns all organizations
+`query getAllUsers` -> Returns all users
 
-`getOrg(orgId)` -> Returns a single organization by ID
+`query getUserById(id: ID!)` -> Returns a single user by ID
 
-`addOrg(org)` -> Returns the created org
+`mutation addUser(input: NewUserInput!)` -> Returns a created user
 
-`updateOrg(orgId)` -> Update an organization by ID
+`mutation updateUser(id: ID!, input: UpdateUserInput!)` -> Update a user by ID
 
-`deleteOrg(orgId)` -> Delete an organization by ID
+`mutation removeUser(id: ID!)` -> Delete a user by ID
 <br>
```

Comment:
We probably don't need these `<br>` tags.

Reply:
Totally agreed! I also thought it seemed a bit redundant, so I removed them.